.travis.yml

```diff
@@ -85,15 +85,15 @@ jobs:
       os: linux
       arch: amd64
       dist: focal
-      go: 1.23.x
+      go: 1.24.x
       script:
         - travis_wait 45 go run build/ci.go test $TEST_PACKAGES
 
     - stage: build
       if: type = push
       os: linux
       dist: focal
-      go: 1.22.x
+      go: 1.24.x
```

Comment:
This is not correct. We always want to keep a builder around for the previous Go releases, to catch regressions where we accidentally start using Go 1.24 features.